site/ko/blog/new-in-devtools-93/index.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: "layouts/blog-post.njk"
+title: "DevTools 업데이트 소식 (Chrome 93)"
+authors:
+  - jecelynyeen
+date: 2021-07-28
+updated: 2021-07-28
+description:
+  "수정 가능한 CSS 컨테이너 쿼리, 웹 번들 프리뷰, Console 에서 더 나은 문자열 제어 및 추가 업데이트 사항"
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/cNXHu1JnHCfRSnIr1bV7.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-93
+---
+
+<!-- Please help to translate the text below to Korean -->
+*Thanks [your name](https://developers.google.com/community/experts/directory/profile/profile-eun-cho) for the translations, []() and []() for the reviews!*
```

Comment:
Please translate this line and put in your preferred name and URL. I just simply copy a url for you at the moment.